pkg/storage/secret/migrator/migrator.go

```diff
@@ -117,6 +118,21 @@ func initSecretStore(mg *migrator.Migrator) string {
 		Indices: []*migrator.Index{},
 	})
 
+	tables = append(tables, migrator.Table{
+		Name: TableNameSecureValueOutbox,
+		Columns: []*migrator.Column{
+			{Name: "uid", Type: migrator.DB_NVarchar, Length: 36, IsPrimaryKey: true}, // Fixed size of a UUID.
+			{Name: "message_type", Type: migrator.DB_TinyInt, Nullable: false},
```

Comment:
Writing here instead of where `type OutboxMessageType int` is declared, but wouldn't it be clearer to have this as a `string`? I know it occupies more storage space, but do we need to optimize for it (knowing the type at a glance seems more valuable)?